drivers/gpu/drm/panel/panel-ft8716u.c

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2023 FIXME
+// Generated with linux-mdss-dsi-panel-driver-generator from vendor device tree:
+//   Copyright (c) 2013, The Linux Foundation. All rights reserved. (FIXME)
+
+#include <linux/delay.h>
+#include <linux/gpio/consumer.h>
+#include <linux/module.h>
+#include <linux/of.h>
+
+#include <drm/drm_mipi_dsi.h>
+#include <drm/drm_modes.h>
+#include <drm/drm_panel.h>
+
+struct ft8716u {
+	struct drm_panel panel;
+	struct mipi_dsi_device *dsi;
+	struct gpio_desc *reset_gpio;
+	bool prepared;
+};
+
+static inline struct ft8716u *to_ft8716u(struct drm_panel *panel)
+{
+	return container_of(panel, struct ft8716u, panel);
+}
+
+#define dsi_dcs_write_seq(dsi, seq...) do {				\
```

Comment:
You should be able to use [standard mipi_dsi_dcs_write_seq](https://elixir.bootlin.com/linux/latest/C/ident/mipi_dsi_dcs_write_seq) instead of your own invented macro